app/helpers/games_helper.rb

```diff
@@ -4,6 +4,11 @@ def render_piece(x, y)
     return nil if piece.blank?
     
     link_to image_tag(piece.icon_path, size: '48x48'), piece_path(piece[:id])
-    if piece == @current_piece 
+    if piece == @current_piece
+      "<div class='selected'>".html_safe
+    else 
+      "<div>".html_safe
+    end
+    "</div>".html_safe
   end
```

Comment:
Hey @mscoutermarsh, please see all gists I've tried the past few days:

[https://gist.github.com/MattW216/50ce5bd38a6f69662a54f207f3e1d5f6](url)
- no quotes around `<div>` and `</div>` caused controller error.

[https://gist.github.com/MattW216/e9d315b95330f1afb7098b34bc55d157](url)
- quotes around `<div>` and `</div>` showed the actual HTML of `</div>` in the squares instead of the pieces.

[https://gist.github.com/MattW216/4dccdbebed4fb8395f20f6b9a2ccd7c2](url)
- This attempt did highlight the square I put the code on, but doesn't seem very efficient to write this out on every square. Also, it would highlight that specific square regardless of what piece you select, which kind of makes sense, since when you see the code in Sublime, it's in yellow (treating like a string, not ruby code), thus not actually processing the conditional.

The code above in this commit is with adding `.html_safe` but completely screws up the board - there are squares all over the screen, and no pieces appear. I tried using `html_safe` and `raw` in different ways in both the helper method in `games_helper.rb` and `pieces#show.html.erb`. No dice 😞

Reply:
@MattW216 Take a look at `content_tag`. I think it will solve your problems.

http://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag

Reply:
@mscoutermarsh Where were you with that a week ago?! 😄